components/turret.py

```diff
@@ -84,6 +101,16 @@ def slew_local(self, angle: float) -> None:
     def get_angle(self):
         return self.motor.getSelectedSensorPosition() / self.COUNTS_PER_TURRET_RADIAN
 
+    @magicbot.feedback
+    def absolue_encoder_reading(self):
+        return constrain_angle(
+            -self.turret_absolute_encoder.getDistance() + self.abs_offset
```

Comment:
Can you make the DutyCycleEncoder do the inversion?

Reply:
There is no explicit method to invert this. I can change the setDistancePerRotation call to use -tau instead of positive tau to handle this correctly if you like?